src/Simplex/Chat/Migrations/M20230922_remote_controller.hs

```diff
@@ -19,7 +19,8 @@ CREATE TABLE remote_hosts ( -- hosts known to a controlling app
 CREATE TABLE remote_controllers ( -- controllers known to a hosting app
   remote_controller_id INTEGER PRIMARY KEY,
   display_name TEXT NOT NULL,
-  fingerprint BLOB NOT NULL
+  fingerprint BLOB NOT NULL,
+  accepted INTEGER
```

Comment:
```suggestion
  accepted INTEGER NOT NULL DEFAULT 0
```

Reply:
It's a tri-state field: non-resolved, resolved as accepted/rejected.